aten/src/ATen/native/ComparisonUtils.cpp

```diff
@@ -14,6 +15,20 @@ class Tensor;
 
 namespace native {
 
+template<typename O, typename C>
+void _assert_match_sym(const O& original, const C& compared, const std::string& name) {
+  if (compared) {
+    auto sym_compared = c10::SymIntArrayRef(
+      reinterpret_cast<const c10::SymInt*>(compared.value().data()), compared.value().size());
+    bool equal = (original == sym_compared);
+    if (!equal) {
+      std::stringstream msg;
+      msg << "Tensor " << name << " mismatch!";
```

Comment:
This stringstream is unnecessary, the `AT_ASSERT` (or more modern `TORCH_CHECK`) macro can handle concatenating string objects together if you comma separate them

Reply:
oh because you copy paste from below. 🤔  why copy paste

Reply:
temp